components/calendar--event.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <li
+    v-popover="{
+      name: firstEvent.id
+    }"
+    class="
+      block list-none mx-1 py-2 border-t border-blue
+      hover:bg-blue-lightest
+    "
+  >
+    <a
+      :href="firstEvent.url"
+      class="w-full block no-underline text-blue-darker"
+      target="_blank"
+      rel="noreferrer noopener"
+    >
+      <span v-if="eventGroup(firstEvent)">
+        <logo
+          v-if="eventGroup(firstEvent)"
+          :icon-set="eventGroup(firstEvent).iconSet"
+          :icon-name="eventGroup(firstEvent).iconName"
+          :icon-text="eventGroup(firstEvent).iconText"
+          class="inline-block"
+        />
+      </span>
+      {{ formatTimeOfEvent(firstEvent) }}
+      <div class="mt-1">{{ eventGroup(firstEvent).name }}</div>
+      <div
+        v-if="additionalEvents.length > 0"
+        class="text-sm">
+        <i>also:
```

Comment:
Can you replace the `<i>` tag with a `<aside class="italic">`? This is more semantically fitting and more clear to screen readers (a11y ftw).

Reply:
Make sense to me. Thanks again.